src/hmem_cuda.c

```diff
@@ -760,6 +760,16 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+int cuda_flush_rdma_operations(void)
+{
+#if (CUDART_VERSION >= 1130)
```

Comment:
You need to use `11030` to specify version 11.3

Reply:
Also, you might be asked to use the `#if` directive to define 2 separate functions instead of inlining the preprocessor logic in this single function.

Reply:
thanks!